pkgs/development/libraries/pipewire/wireplumber.nix

```
@@ -66,12 +67,15 @@ stdenv.mkDerivation rec {
       ))
     ];
 
-  buildInputs = [
-    glib
-    systemd
-    lua5_4
-    pipewire
-  ];
+  buildInputs =
+    [
+      glib
+    ]
+    ++ lib.optional withSystemd systemd
+    ++ [
+      lua5_4
+      pipewire
+    ];
```

```suggestion
      lua5_4
      pipewire
    ]
    ++ lib.optional withSystemd systemd;
```

It feels odd to have a conditional smacked right on the middle of a list...

---

Would you approve the PR if it wasn't and caused a rebuild?

---

Ah, you're right 😀

---

I switched to staging so inputs can be reordered.